components/developer/rust/Makefile

```diff
@@ -58,9 +58,12 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+#omnios uses gcc 7.
+GCC_VERSION =  7
+
 PATH=$(GCC_BINDIR):$(PATH.gnu)
 
-PATCH_LEVEL=0
+PATCH_LEVEL=1
```

Comment:
Please, remove GCC_VERSION=7 as it's already default one, the same goes for PATCH_LEVEL=1

Reply:
Done